cli/src/usage.mjs

```diff
@@ -41,6 +41,8 @@ const lines = [
   { cmdInfo: "--broadcastEvent ../examples/device-onDeviceNameChanged1.event.json",       comment: "Send BroadcastEvent (method, result keys expected)" },
   { cmdInfo: "--sequence ../examples/events1.sequence.json  ",                            comment: "Send an event sequence (See examples/device-onDeviceNameChanged.sequence.json)" },
   { cmdInfo: "--session start/stop  ",                                                    comment: "Start/Stop Firebolt session recording" },
+  { cmdInfo: "--sessionOutput log|raw|mock-overrides  ",                                  comment: "Set the output format to; log: (paired time sequence of calls, responses)|raw: similiar to log but not paired with request|mock-overrides: a directory of mock overrides" },
+  { cmdInfo: "--sessionOutputPath ../examples/path  ",                                    comment: "Specifiy the session output path. Default for 'log' format will be ./sessions and ./mocks/<START_TIME> for 'mock-overrides'." },
```

Comment:
1) do you mean a sessions subdirectory of the current directory whatever it is, or do you mean <repoRoot>/sessions? Same q for mocks?

2) If these are within the repo, let's be sure to add them to .gitignore... we don't want to check these in.

3) Can/should we have some kind of "output" directory above any/all of these kinds of files (so there'd just be one entry in gitignore and any kind of generated files would all be "in one place"?

Reply:
1. Yes so the default would be a subdirecotry in mock-firebolt/server/
2. Sessions was already added, added mocks as well
3. Will modify to be output/(sessions|mocks)